src/zampy/datasets/converter.py

```diff
@@ -0,0 +1,81 @@
+"""Dataset formatter for different conventions."""
+import json
+from pathlib import Path
+from typing import Union
+import cf_xarray.units  # ruff: noqa: F401
+import pint_xarray  # ruff: noqa: F401
+import xarray as xr
+
+
+CONVENTIONS = {
+    "ALMA": "src/zampy/conventions/ALMA.json",
+}
+
+
+def check_convention(convention):
+    """Check if the given convention is supported."""
+    if convention.upper() not in CONVENTIONS:
+        raise ValueError(
+            f"The '{convention}' convention is not supported.\n"
+            "Please check the available conventions in the `conventions` "
+            "directory and choose one from there."
+        )
+    else:
+        print(f"Start converting data to follow the '{convention}' convention.")
+
+
+def convert(
+    dataset: xr.Dataset, fname: str, convention: Union[str, Path]
```

Comment:
I tried to put 'from zampy.datasets.dataset_protocol import Dataset' into `src/zampy/__init__.py` and then import `zampy.Dataset`, but it ends up with a circular dependency error. 

Simply put `from zampy.datasets.dataset_protocol import Dataset` in `converter.py` works nicely, but then it becomes `data: xr.Dataset, dataset: Dataset, convention: Union[str, Path]`, which is less clear. Let me know if you have a different solution. Otherwise we could just keep it this way.